src/stdJson.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+
+import "Vm.sol";
+
+// Helpers for parsing keys into types.
+library JsonParser {
+
+    address constant private VM_ADDRESS =
+        address(bytes20(uint160(uint256(keccak256('hevm cheat code')))));
+
+    Vm internal constant vm = Vm(VM_ADDRESS);
```

Comment:
This is to make the library `v0.3`-ready. We use this approach for all components on that brach.

```suggestion
    Vm private constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
```